torch/nn/modules/module.py

```diff
@@ -564,11 +583,6 @@ def state_dict(self, destination=None, prefix='', keep_vars=False):
         (rather than a Tensor).
 
         Args:
-            destination (dict, optional):
-                if not None, the return dictionary is stored into destination.
-                Default: None
-            prefix (string, optional): Adds a prefix to the key (name) of every
-                parameter and buffer in the result dictionary. Default: ''
             keep_vars (bool, optional): if ``True``, returns a Tensor for each
```

Comment:
This one should be private too. The message is really funny now that we don't have Variables 😄